### PR TITLE
RELATED: ONE-4070 Document new onApplyWithFilterDefinition callback

### PR DESCRIPTION
**Pull request by ivan-nejezchleb**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/122/ : SUCCESS in 1m 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/10/ : SUCCESS in 13s

File: docs/attribute_filter_component.md [hunk 1]
```diff
@@ -64,12 +64,44 @@ To define the attribute values that should be selected in the filter by default,
...
+            this.setState(filter);
+        }
+
+        render() {
+            return (
+                <div>
+                    <AttributeFilter
+                        filter={this.state.filter}
+                        projectId={projectId}
+                        onApply={this.onApplyWithFilterDefinition}
```
Comment (lupko) [inline docs/attribute_filter_component.md]: should the propname be 'onApplyWithFilterDefinition' instead of onApply?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/123/ : SUCCESS in 1m 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/11/ : SUCCESS in 6s

File: docs/attribute_filter_component.md (hunk 1 above)
Comment (ivan-nejezchleb) [reply docs/attribute_filter_component.md]: of course, thx

Comment (evgeniabogranskaya) [reply docs/attribute_filter_component.md]: fixed

**Comment by evgeniabogranskaya** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/124/ : SUCCESS in 3m 11s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/12/ : SUCCESS in 45s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/125/ : SUCCESS in 2m 30s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/13/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/126/ : SUCCESS in 39s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/37/ : SUCCESS in 16s

**Pull request merged**